README.md

```diff
@@ -35,16 +35,10 @@ To register/resolve the identifier, do the following:
 import { composeAPI } from '@tangleid/core';
 
 const tid = composeAPI({
-  providers: {
-    // mainnet
-    '0x1': 'http://node.deviceproof.org:14265',
-    // devnet
-    '0x2': 'https://nodes.devnet.thetangle.org:443',
-  },
+  provider: 'http://node.deviceproof.org:14265',
```

Comment:
Can we unify to `https://tangle.puyuma.org`? The deviceproof is likely being retired soon.

Reply:
Modified to the community version of IRI load balancer `https://tangle.puyuma.org`.